fastdds_monitor.repos

```diff
@@ -2,28 +2,28 @@ repositories:
     foonathan_memory_vendor:
         type: git
         url: https://github.com/eProsima/foonathan_memory_vendor.git
-        version: master
+        version: v1.2.0
     fastcdr:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
-        version: master
+        version: v1.0.23
     fastdds:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
-        version: master
+        version: v2.5.0
     fastdds_statistics_backend:
         type: git
         url: https://github.com/eProsima/Fast-DDS-statistics-backend.git
-        version: main
+        version: v0.4.0
     fastrtpsgen:
         type: git
         url: https://github.com/eProsima/Fast-DDS-Gen.git
-        version: master
+        version: v2.1.0
     fastrtpsgen/thirdparty/idl-parser:
         type: git
         url: https://github.com/eProsima/IDL-Parser.git
```

Comment:
Do we really need this in the Monitor?

Reply:
Rebase over https://github.com/eProsima/Fast-DDS-monitor/pull/141 to remove them